akka-actor/src/main/resources/reference.conf

```diff
@@ -134,7 +134,7 @@ akka {
     unstarted-push-timeout = 10s
 
     typed {
-      # Default timeout for typed actor methods with non-void return type
+      # Default timeout for the depredated TypedActor (not the new actor APIs in 2.6) methods with non-void return type
```

Comment:
I added similar clarification in https://github.com/akka/akka/pull/28340/files#diff-618d4a56737cdd72d358885898017a3dR137

Both can go in.